push_reference_update.py

```diff
@@ -0,0 +1,128 @@
+"""
+Script file for generating reference updates
+"""
+
+from idseq_pipeline.commands.common import *
+
+local_work_dir = "idseq_pipeline_temp"
+ncbitool_s3_path = "s3://czbiohub-infectious-disease/ncbitool"
+
+
+def main():
+    ##### PARAMETERS TO EDIT #####
+
+    ## To pull NCBI references through ncbitool, which archives and records versions, set URL_PREFIX=''.
+    # To pull the latest files from NCBI without using ncbitool (no version information), set URL_PREFIX=ftp://ftp.ncbi.nlm.nih.gov.
+    os.environ["URL_PREFIX"] = "ftp://ftp.ncbi.nlm.nih.gov"
```

Comment:
we can make this 
os.environ["URL_PREFIX"] = os.environ.get("URL_PREFIX", "ftp://ftp.ncbi.nlm.nih.gov") 
and ditto for the other env in case we want to overwrite them.